API.md

```diff
@@ -808,6 +810,24 @@ const boolean = Joi.boolean();
 boolean.validate(true, (err, value) => { });
 ```
 
+#### `boolean.truthy(value)`
+
+Allows for additional strings to be considered valid boolean values in addition to default 'truthy' strings: ['true', 'yes', '1', 'on']. Accepts a string or an array of strings.
```

Comment:
You say `additional` here, but it doesn't appear to be additional (and I personally don't think it should be). Instead something like `Allows you to specify the string values considered to be valid truthy boolean values...` or something similar. 

The same goes for the `falsey` one below.